schemas/v0.1/sample.yml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=./yaml.schema.json
+
+version: v0.1
+
+workingDir: ai-project/	# default artifacts folder
+
+environmentFolder: ai-project/env/	# .env files location
+
+dataset:        # dataset feature definition
+  content:      # recognizable dataset files
+    - dataset/output/*.json 	# glob pattern
+
+evaluation:     # evaluation feature definition
+  result:       # recognizable evaluation results
+    - evaluation/result/*.json	# glob pattern
+
+
+finetuning: # finetuning feature definition
+  runOn: containerapp # running on Azure Container Apps (ACA)
+  provision:  # provision step definition
+    subscriptionId: ${{AZURE_SUBSCRIPTION_ID}}
+    resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}}
+    templates:
+      - path: infra/finetuning.bicep
+        parameters: infra/finetuning.parameters.json
```

Comment:
Do we need the output variable name mapping which is similar to Teams Toolkit?

Reply:
Bicep outputs are directly to variable, similar to TTK's `arm/deploy`, so no mapping need for provision.